test/e2e/components/vl-form-validation.js

```diff
@@ -1,6 +1,37 @@
 const { VlElement } = require('vl-ui-core').Test;
 
+
 class VlFormValidation extends VlElement {  
+       
+    async isRequired() {
+        return this.hasAttribute('data-required');
+    }
+
+    async getErrorMessage() {
+        return this.getAttribute('data-vl-error-message');
+    }
```

Comment:
Eventueel kon hier ook gezocht worden naar een sibling met attribuut `is='vl-form-validation-message'`, maar misschien overkill dus goed zoals het nu is.